README.md

```diff
@@ -0,0 +1,15 @@
+# Bible-Trivia-Plus
+
+A Bible trivia game.
+
+## Prerequisites
+
+A webbrowser.
+
+## How to install the game
+
+Clone the repository, open the index.html file in a browser.
+
+## State of the program
+
+As of yet, only Torah questions can be played.
```

Comment:
This really isn't playable at all yet: the questions are just samples, they're not tagged properly, there's no scoring or goal, etc.